applications/nrf_desktop/doc/watchdog.rst

```diff
@@ -0,0 +1,43 @@
+.. _watchdog:
+
+Watchdog module
+###############
+
+The ``watchdog`` module is responsible for controlling the watchdog timer.
+The watchdog timer is used to prevent hangs caused by software or hardware faults.
+When the system hangs, it is automatically restarted by the watchdog timer.
+
+Module Events
+*************
+
++----------------+-------------+--------------+-----------------+------------------+
+| Source Module  | Input Event | This Module  | Output Event    | Sink Module      |
++================+=============+==============+=================+==================+
+|                |             | ``watchdog`` |                 |                  |
++----------------+-------------+--------------+-----------------+------------------+
+
+External dependencies
+*********************
+
+The module uses the Zephyr :ref:`zephyr:watchdog` driver.
+For this reason you should set :option:`CONFIG_WATCHDOG` option.
+
+Configuration
+*************
+
+The module is enabled by the ``CONFIG_DESKTOP_WATCHDOG_ENABLE`` option.
+
+You must define ``CONFIG_DESKTOP_WATCHDOG_TIMEOUT``.
+After this amount of time (in ms), the device will be restarted if the watchdog timer was not fed.
```

Comment:
Rephrase `was not fed` to something more specific.

Reply:
ok